src/components/todo-item/Index.module.css

```diff
@@ -0,0 +1,91 @@
+.container {
+  padding: 8px, 16px, 8px, 16px;
+  gap: 12px;
+  margin-top: 5px;
+  margin-bottom: 5px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  position: relative;
+  width: 440px;
```

Comment:
The width of this element should not be defined. It should shrink/grow with it parent component.

So leave the width to be 100%, and make sure the parent layout will define the width.